tests/test_utils.py

```diff
@@ -86,6 +86,116 @@ def test_parse_json_with_comments():
     eq_(result, {'key1': 'value1', 'key2': 'value2', 'key3': 5})
 
 
+def test_parse_json_with_comments_single_line_comments():
+
+    # Need to add comments
+    json_with_comments = ("""{"key1": "value1", //some comments \n"""
+                          """//more comments \n"""
+                          """"key2": "value2", "key3": 5}""")
+
+    tempf = tempfile.NamedTemporaryFile(mode='w', suffix='.csv', delete=False)
+    filename = tempf.name
+    tempf.close()
```

Comment:
I could be wrong but I think you can just use `tempf` in place of `buff` below, can't you? That is, I think you can get rid of this line and the `with` part below.

```
❯ ~/opt/python/3.8/bin/ipython
import tempfile
Python 3.8.1 | packaged by conda-forge | (default, Jan 29 2020, 15:06:10)
Type 'copyright', 'credits' or 'license' for more information
IPython 7.12.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import tempfile

In [2]: json_with_comments = ("""{"key1": "value1", //some comments \n"""
   ...:                           """//more comments \n"""
   ...:                           """"key2": "value2", "key3": 5}""")

In [3]: tempf = tempfile.NamedTemporaryFile(mode='w', suffix='.csv', delete=False)

In [4]: filename = tempf.name

In [5]: tempf.write(json_with_comments)
Out[5]: 81

In [6]: tempf.close()

In [7]: filename
Out[7]: '/var/folders/f_/_561m4hx5cz3tn5hj63ryvjxvw9dhw/T/tmpc4cithhk.csv'

In [8]: ! cat /var/folders/f_/_561m4hx5cz3tn5hj63ryvjxvw9dhw/T/tmpc4cithhk.csv
{"key1": "value1", //some comments
//more comments
"key2": "value2", "key3": 5}
```

Reply:
Sounds good! I'll make that change!